collection/screenshot/capture-screenshot.yml

```diff
@@ -22,7 +22,9 @@ rule:
           - api: user32.GetWindowDC
           - api: user32.GetDC
           - and:
-            - api: gdi32.CreateDCA
+            - or:
+              - api: gdi32.CreateDCA
+              - api: gdi32.CreateDCW
```

Comment:
```suggestion
            - api: gdi32.CreateDC
```

Reply:
capa automatically extracts API features with the A/W truncated

Reply:
🤦🏻